### PR TITLE
Move aria label util into module

### PR DESCRIPTION
**Pull request by hghazzi**
To better break up the UI logic from the rest of the player, move aria labels into its own module. 

JW7-4282

### CODE REVIEW

**Comment by johnBartos** [comment]
Use the PR template

File: src/js/controller/aria.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+        element.setAttribute('aria-label', ariaLabel);
+        element.setAttribute('role', 'button');
+        element.setAttribute('tabindex', '0');
+    };
+
+    return {
+        addLabel: addLabel
+    };
+});
```
Comment (robwalch) [inline src/js/controller/aria.js]: Put this in the utils directory. It's not a controller / has no business logic.

File: src/js/view/controls/components/tooltip.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 define([
     'utils/backbone.events',
+    'controller/aria',
```
Comment (robwalch) [inline src/js/view/controls/components/tooltip.js]: 'utils/aria'

File: src/js/utils/aria.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        }
+
+        element.setAttribute('aria-label', ariaLabel);
+        element.setAttribute('role', 'button');
+        element.setAttribute('tabindex', '0');
+    };
+
+    return {
+        addLabel: addLabel
+    };
```
Comment (robwalch) [inline src/js/utils/aria.js]: Just return the function. Waste of an object and `. addLabel` can't be minified.

**Pull request merged**